Frontend/src/app/views/dashboard/profile/profile.component.scss

```diff
@@ -44,6 +46,10 @@ button {
 
 /* Media query for tablet viewport targets screen size with a minimum width of 660px. */
 @media only screen and (min-width: 660px) {
+  .container {
+    padding: 40px;
+  }
+
```

Comment:
Can probably just remove this and use the padding that the shared `.container` gives